code/_onclick/ai.dm

```diff
@@ -139,6 +139,7 @@
 
 	toggle_bolt(usr)
 	add_hiddenprint(usr)
+	ui_update()
```

Comment:
Ui update is handled in toggle bo

Reply:
That's not the case as far as I can tell, unless you mean it should be handled in toggle_bolt? It updates icon and sends a signal, but I don't see ui_update or similar there.

Reply:
I moved the call deeper into `set_bolt`, leaving this open in case there is in fact another `ui_update` I haven't noticed